node/src/chain_spec/turing.rs

```diff
@@ -106,9 +106,9 @@ pub fn turing_development_config() -> ChainSpec {
 						XcmFlow::Normal,
 					),
 					(
-						2000,
-						turing_runtime::NATIVE_TOKEN_ID,
-						false,
+						2000, // Shibuya
+						4,    // SBY Currency ID
+						true,
```

Comment:
Will need to make this change through governance on turing and turing-staging. Note: 4 is specific to dev. Will need to look up the appropriate currency id in deployed envs.